server/src/main/java/org/elasticsearch/transport/ConnectionProfile.java

```diff
@@ -134,6 +134,7 @@ public static ConnectionProfile buildSingleChannelProfile(
     private final TimeValue pingInterval;
     private final Compression.Enabled compressionEnabled;
     private final Compression.Scheme compressionScheme;
+    private final String authorization;
```

Comment:
Is this necessary here ?  Unless there is a follow up that uses the value from here, I think we should omit it from here. It is not clear if this object needs access to the authorization and if in doubt keep it out.

Reply:
I can remove from strategy and profile when I know where to move it. If authorization does not belong there, what class does it belong in?

https://github.com/elastic/elasticsearch/pull/90312#discussion_r983138025

Reply:
Removed authorization from both the profile and strategy classes.